src/Page/TextLayer.spec.tsx

```diff
@@ -161,42 +179,7 @@ describe('TextLayer', () => {
       const wrapper = container.firstElementChild as HTMLDivElement;
       const textItems = wrapper.children;
 
-      expect(textItems).toHaveLength(desiredTextItems.length + 1);
-    });
-
-    it('maps textContent items to actual TextLayer children properly', async () => {
```

Comment:
Personally I feel this test is pointless now as there is no longer a direct mapping between customTextRender and normal.